config/locales/in_person_proofing/en.yml

```diff
@@ -9,32 +9,38 @@ en:
       barcode:
         cancel_link_text: Cancel your barcode
         close_window: You may now close this window.
-        deadline: Visit the Post Office by %{deadline}.
-        deadline_restart: If you go after the deadline, your information will not be
+        deadline: You must visit any participating Post Office by %{deadline}.
+        deadline_restart: If you go after this deadline, your information will not be
           saved and you will need to restart the process.
-        email_sent: We have sent this information to the email you used to log in.
-        items_to_bring: 'Show the following items to any Post Office associate:'
+        email_sent: We have sent this information to the email you used to sign in.
         learn_more: Learn more about what to bring
         location_details: Location details
-        questions: Question?
+        questions: Questions?
         retail_hours: Retail hours
         retail_hours_closed: Closed
-        return_to_partner_html: You may now close this window or %{link} to complete any
-          next steps you can access until your identity has been verified.
-        return_to_partner_link: return to %{sp_name}
-        speak_to_associate: You can see any Post Office associate at the retail window
-          to verify your identity. No appointment is required.
+        return_to_partner_html: You may now %{link} to complete any next steps you can
```

Comment:
I'm curious why the screenshot doesn't show the `return_to_partner.html` text?

PR screenshot: 
<img width="470" alt="image" src="https://user-images.githubusercontent.com/12240896/234407162-1293a647-c141-4768-805b-1225a551c4de.png">

Figma page screenshot (correct in the code file):
<img width="277" alt="image" src="https://user-images.githubusercontent.com/12240896/234407256-46ec2aeb-247d-412f-be02-01a045a51a6f.png">

Reply:
Ah, good catch - I forgot to come back to that in my testing after all the translation stuff... what I have shown is the default if there is no "Partner Agency" detected.  How do the Joy Engineers do testing with a Partner Agency?  

(For others out side of Team Joy:  All of my attempts to use the Agnes Sinatra App for Partner Agency testing do not allow me in to the IPP flow 😕)

Reply:
@kellular, Here is an example where the Partner Agency was present: 
![image](https://user-images.githubusercontent.com/13384921/234660798-affe0ecc-b011-4b3a-8a11-786fa58024b1.png)